tests/integration/mixer/k8s/k8s_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mixer
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/label"
+)
+
+func TestK8sDeployment(t *testing.T) {
+	// Test to ensure that deployment in K8s environment completes. We can extend this suite in the future
+	// to include more Mixer/K8s integration tests. In that case, it may make sense to move the deployment
+	// logic into TestMain.
+	framework.NewTest(t).Run(func(ctx framework.TestContext) {
+		cfg := istio.DefaultConfigOrFail(ctx, ctx)
+		cfg.Values["global.useMCP"] = "false"
+		cfg.Values["galley.enabled"] = "false"
+		cfg.DoNotWaitForValidationWebhook = true
+
+		_, err := istio.Deploy(ctx, &cfg)
```

Comment:
Does this check that crashloops don't happen? What concerns me is that even in a regular install with Galley, Mixer crash loops in the beginning. So how do you tell a difference between intermittent crashes and hard crash without Galley?

Reply:
IMHO we should run a basic Mixer-dependent functionality test to be sure it's actually working.

Reply:
It doesn't check for crashloops. It does ensure that all components come up (liveness & readiness) eventually.

Reply:
Sounds good, but I didn't want to copy paste some arbitrary test here, and I don't have the bandwidth to come up with a good factoring or a representative test. If you have a cheap recommendation, I am happy to incorporate.

Reply:
There is a delay in the liveness/readiness probe signal showing up. I think it's on the order of 30s before k8s realizes it's dead. Maybe insert a 5 minutes delay and check for readiness after that?

Reply:
Nice catch. I added additional delay/checks. PTAL.